Godeps/Godeps.json

```diff
@@ -504,18 +504,6 @@
 			"Rev": "f368244301305f414206f889b1735a54cfc8bde8"
 		},
 		{
-			"ImportPath": "github.com/spf13/cobra/cobra",
```

Comment:
`github.com/spf13/cobra/cobra` and `github.com/spf13/cobra/cobra/cmd` don't seem to be imported from anywhere even from our vendored libraries, and they're also a different version.  This might be causing Godep issues.

`github.com/spf13/cobra/doc` just doesn't seem to be used anywhere.